src/views/TransactionBuilder.js

```diff
@@ -95,6 +95,7 @@ function onAttributeUpdate(dispatch, param, value) {
     case "memo":
       newAttributes.memoType = value.type;
       newAttributes.memoContent = value.content;
+      break;
```

Comment:
❔ Would it be realistic to write a test for this? I see that tests are pretty light already in this repo, so maybe there isn't the framework for it.

Reply:
Tests would have definitely caught this bug. 💯 We are adding tests as we keep refactoring, but we haven't reached the `TransactionBuilder` view yet. Hopefully, we'll have time to get to it soon. Tier 2 progress can be found [here](https://github.com/stellar/laboratory/issues/557).